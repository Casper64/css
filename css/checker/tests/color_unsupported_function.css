.other {
    color: bla(255, 255, 255);
}