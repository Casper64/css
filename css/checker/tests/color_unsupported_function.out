css/checker/tests/color_unsupported_function.css:2:12: error: unsupported function "bla" for property "color".
Did you mean `rgb`, `rgba`, `hsl`?
    1 | .other {
    2 |     color: bla(255, 255, 255);
      |            ~~~~~~~~~~~~~~~~~~
    3 | }